### ISSUE TITLE
sysctl_file is broken if file doesn't exists

### ISSUE THREAD

**Issue by evgkrsk** [ansible/ansible#6482]
##### Issue Type:

Bug Report
##### Ansible Version:

ansible 1.5
##### Environment:

Running from OS: ALT Linux Sisyphus (current)
Managed OS: ALT Linux Sisyphus (current)
##### Summary:

Module sysctl is broken if sysctl_file is not pre-created.
##### Steps To Reproduce:

```
ansible buster -m sysctl -a 'name=kernel.perf_event_paranoid value=2 sysctl_file=/tmp/ansible.conf'
```
##### Expected Results:

```
buster | success >> {
    "changed": true
}
```
##### Actual Results:

```
buster | FAILED => failed to parse: Traceback (most recent call last):
  File "/home/teer/.ansible/tmp/ansible-tmp-1394774220.58-41805730407451/sysctl", line 1381, in <module>
    main()
  File "/home/teer/.ansible/tmp/ansible-tmp-1394774220.58-41805730407451/sysctl", line 304, in main
    result = SysctlModule(module)    
  File "/home/teer/.ansible/tmp/ansible-tmp-1394774220.58-41805730407451/sysctl", line 113, in __init__
    self.process()
  File "/home/teer/.ansible/tmp/ansible-tmp-1394774220.58-41805730407451/sysctl", line 131, in process
    self.read_sysctl_file()
  File "/home/teer/.ansible/tmp/ansible-tmp-1394774220.58-41805730407451/sysctl", line 229, in read_sysctl_file
    lines = open(self.sysctl_file, "r").readlines()
IOError: [Errno 2] No such file or directory: '/tmp/ansible.conf'
```

**Comment by evgkrsk** [comment]
Module works fine with pre-created sysctl_file:

```
teer@teer ~ $ansible buster -m shell -a 'touch /tmp/ansible.conf'
buster | success | rc=0 >>               


teer@teer ~ $ansible buster -m sysctl -a 'name=kernel.perf_event_paranoid value=2 sysctl_file=/tmp/ansible.conf'
buster | success >> {       
    "changed": true
}

teer@teer ~ $ansible buster -m shell -a 'rm -f /tmp/ansible.conf'                                               
buster | success | rc=0 >>
```

**Comment by jctanner** [comment]
I will take care of this.

**Comment by jctanner** [comment]
@evgkrsk this should be fixed in devel by 9d6518de5a9e76aed8b7ec3b79c4ae693c1caa6c
